pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -384,6 +393,19 @@ func (h *NetworkUtilsHandler) StartDHCP(nic *VIF, serverAddr net.IP, bridgeInter
 	return nil
 }
 
+func (h *NetworkUtilsHandler) CreateNDPConnection(bridgeInterfaceName string, launcherPID int) error {
```

Comment:
I would change to `CreateAndExportNDPConnection`

Reply:
https://github.com/kubevirt/kubevirt/pull/4581/commits/014fadcf3e4d807d9a21db6cbfa3dc3f96bdaa07?file-filters%5B%5D=.go#diff-7cea0cc72a89ff86def413e0982f49cbbb527157a092cd507a336ced23b7ce2aR390